envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/groups/AccessLogFilterFactory.kt

```diff
@@ -0,0 +1,51 @@
+package pl.allegro.tech.servicemesh.envoycontrol.groups
```

Comment:
Actually, because it has two responsibilities:
- parsing
- producing envoy specific config

I'd split into two classes. The parsing one can stay here I guess. Or be moved to NodeMetadata class' file.

Reply:
moved to another scope and splited functionality